src/components/OnboardingMenu/components/OnboardingMenuItem/OnboardingMenuItem.tsx

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+
+import {CircleCheckFill, Clock} from '@gravity-ui/icons';
+import {ArrowToggle, Disclosure, Icon, Skeleton} from '@gravity-ui/uikit';
+
+import {block} from '../../../utils/cn';
+
+import './OnboardingMenuItem.scss';
+
+export type OnboardingMenuItemProps = {
+    title: string;
+    status?: 'completed' | 'pending';
+    children?: React.ReactNode;
+    buttons?: React.ReactNode;
+    hasDivider?: boolean;
+    loading?: boolean;
+
+    onClick?: () => void;
+};
+const noopFunc = () => {};
+
+const ICON_SIZE = 20;
+
+const cnOnboardingMenuItem = block('guide-menu-item');
+
+export const OnboardingMenuItem = ({
+    title,
+    status,
+    children,
+    buttons,
+    hasDivider = true,
+    onClick = noopFunc,
+    loading = false,
+}: OnboardingMenuItemProps) => {
+    const icon = React.useMemo(() => {
+        if (status === 'pending') {
+            return (
+                <Icon size={ICON_SIZE} data={Clock} className={cnOnboardingMenuItem('time-icon')} />
+            );
+        }
+
+        if (status === 'completed') {
+            return (
+                <Icon
+                    size={ICON_SIZE}
+                    data={CircleCheckFill}
+                    className={cnOnboardingMenuItem('time-icon')}
+                />
+            );
+        }
+
+        return (
+            <Icon
+                size={ICON_SIZE}
+                data={CircleCheckFill}
+                className={cnOnboardingMenuItem('time-icon__disable')}
+            />
+        );
+    }, []);
+
+    const summary = React.useMemo(() => {
+        return <span className={cnOnboardingMenuItem('title')}>{title}</span>;
+    }, []);
+
+    const header = React.useMemo(
+        () => (
+            <div className={cnOnboardingMenuItem('collapse-header')} role="button" tabIndex={0}>
```

Comment:
Disclosure is already interactive element, we cant nest them, also tabIndex makes this element to be focusable, but no handlers are passed